cli/cmd/instances.go

```diff
@@ -30,7 +30,7 @@ func internalInstancesModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 		return errNoConfig
 	}
 
-	err := list.ListInstances(cmdCtx, cliOpts)
+	err := binary.ListInstances(cmdCtx, cliOpts)
```

Comment:
This is not binary-related functionality. Probably, the better place for listing instances is `running` package.